pkg/frontend/authenticate.go

```diff
@@ -1264,6 +1265,8 @@ const (
 	getAccountIdAndStatusFormat = `select account_id,status from mo_catalog.mo_account where account_name = '%s';`
 	getPubInfoForSubFormat      = `select database_name,all_account,account_list from mo_catalog.mo_pubs where pub_name = "%s";`
 	getDbPubCountFormat         = `select count(1) from mo_catalog.mo_pubs where database_name = '%s';`
+
+	fetchSqlOfSpFormat = `select body, args from mo_catalog.mo_stored_procedure where name = '%s' and db = '%s';`
```

Comment:
can you use prepare and bind?   Did you checked if this fmt.Sprintf will cause sql injection?

Wait -- I think all the sql above have the same problem.    @daviszhen @ouyuanning ?